patches/04_dd92364be67a14170bd1ea3575ed1569745e8883.patch

```diff
@@ -0,0 +1,32 @@
+From dd92364be67a14170bd1ea3575ed1569745e8883 Mon Sep 17 00:00:00 2001
+From: nik gaffney <nik@fo.am>
+Date: Sun, 25 Sep 2022 22:30:39 +0200
+Subject: [PATCH] added homebrew arm64 paths
+
+---
+ samplebrain.pro | 5 ++---
+ 1 file changed, 2 insertions(+), 3 deletions(-)
+
+diff --git a/samplebrain.pro b/samplebrain.pro
+index 58ec2bc..f64772b 100644
+--- a/samplebrain.pro
++++ b/samplebrain.pro
+@@ -43,7 +43,8 @@ SOURCES += app/MainWindow.cpp \
+ 
+ INCLUDEPATH += brain/src
+ INCLUDEPATH += /usr/local/include
+-LIBS += -L.. -L/usr/local/lib -lportaudio -lfftw3 -lsndfile -llo -ldl -lpthread -lm
++INCLUDEPATH += /opt/homebrew/include
++LIBS += -L.. -L/usr/local/lib -L/opt/homebrew/lib -lportaudio -lfftw3 -lsndfile -llo -ldl -lpthread -lm
```

Comment:
why do you need this ? There is no `/opt`

Reply:
This one is also a commit directly from upstream. I did it like this so I could apply commits as patches instead of making my own custom ones to keep it closer to upstream. 

I could make a new patch for 6 that doesn't rely on this one if its necessary.